### ISSUE TITLE
Client delete takes &str, but deleting a rune page via id needs String

### ISSUE THREAD

**Issue by AlsoSylv** [Leastrio/Shaco#1]
As the title says, something like 
```rust
	let client = rest::RESTClient::new()?;
	client.delete("/lol-perks/v1/pages/id").await?;
```
expects the page to be delete via the page ID, which requires the use of a String such as 
```rust
	let rune_page_endpoint: String = format!("/lol-perks/v1/pages/{}", id);
```
But this causes an error when passing it as the paramater for `client.delete`

**Comment by Leastrio** [comment]
I'll Look into changing this later today, I believe I did this because of something with reqwest.

**Comment by Leastrio** [comment]
Just published shaco 0.5.0, which has all endpoint params switched to String.
